src/credentials/DeviceAttestationConstructor.cpp

```diff
@@ -123,7 +112,7 @@ CHIP_ERROR DeconstructAttestationElements(const ByteSpan & attestationElements,
 
     VerifyOrReturnError(error == CHIP_NO_ERROR || error == CHIP_END_OF_TLV, error);
 
-    VerifyOrReturnError(certificationDeclarationExists && attestationNonceExists && timestampExists,
```

Comment:
You removed too much. This condition is important and no longer done. We need the `xxxxEsists` bools, just the `VerifyOrReturnError(timestampExists == false, CHIP_ERROR_IM_MALFORMED_COMMAND_DATA_ELEMENT);` lines that checked for duplicate are actually impossible.

Reply:
Addressed in https://github.com/project-chip/connectedhomeip/pull/10546/commits/421e3cd874cbd2c15cd31019e7358bca688f9303